instance.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_ami" "amzn-ami" {
+	most_recent = true
+	filter{
+	  name = "name"
+	  values = ["amzn2-ami-hvm-*-gp2"]
+	}
+	
+	owners = ["137112412989"] #AWS
+
+}
+
```

Comment:
Create `aws_key_pair`. Follow https://www.terraform.io/docs/providers/aws/r/key_pair.html

Make `public_key = var.authorized_key`

create a variable block 
```
variable authorized_key {
   description = "The public ssh rsa key you generated"
}
```
Make sure you add TF_VAR_authorized_key in circleci env variable.